file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [31-May-2025 21:29:30 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>ultimate-email-validator</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[31-May-2025 21:29:30 UTC] PHP Deprecated:  Required parameter $query_args follows optional parameter $action in /wordpress/wp-content/plugins/ultimate-email-validator/admin/includes/admin-components.php on line 769
[31-May-2025 21:29:30 UTC] PHP Deprecated:  Required parameter $element_id follows optional parameter $start_with in /wordpress/wp-content/plugins/ultimate-email-validator/admin/includes/admin-components.php on line 1034
[31-May-2025 21:29:41 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>ultimate-email-validator</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[31-May-2025 21:29:41 UTC] PHP Deprecated:  Required parameter $query_args follows optional parameter $action in /wordpress/wp-content/plugins/ultimate-email-validator/admin/includes/admin-components.php on line 769
[31-May-2025 21:29:41 UTC] PHP Deprecated:  Required parameter $element_id follows optional parameter $start_with in /wordpress/wp-content/plugins/ultimate-email-validator/admin/includes/admin-components.php on line 1034
[31-May-2025 21:29:50 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>ultimate-email-validator</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[31-May-2025 21:29:51 UTC] PHP Deprecated:  Required parameter $query_args follows optional parameter $action in /wordpress/wp-content/plugins/ultimate-email-validator/admin/includes/admin-components.php on line 769
[31-May-2025 21:29:51 UTC] PHP Deprecated:  Required parameter $element_id follows optional parameter $start_with in /wordpress/wp-content/plugins/ultimate-email-validator/admin/includes/admin-components.php on line 1034
[31-May-2025 21:31:28 UTC] PHP Warning:  Undefined array key "DOCUMENT_ROOT" in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:28 UTC] PHP Warning:  include_once(/wp-config.php): Failed to open stream: No such file or directory in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:28 UTC] PHP Warning:  include_once(): Failed opening '/wp-config.php' for inclusion (include_path='.:') in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:28 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SHOW TABLES LIKEwp_schemaLocalBusiness</p>
<p>Queries made or created this session were:</p>
<ol>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Failed to parse the MySQL query.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(606): WP_SQLite_Driver->new_driver_exception('Failed to parse...')
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SHOW TABLES LIK...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SHOW TABLES LIK...')
#3 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SHOW TABLES LIK...')
#4 /wordpress/wp-content/plugins/xenial/xenial-WP.php(79): wpdb->get_var('SHOW TABLES LIK...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): xwp_activation(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_xenial...', false)
#9 /internal/eval.php(20): activate_plugin('xenial/xenial-W...')
#10 {main}</pre>
 for query SHOW TABLES LIKEwp_schemaLocalBusiness made by activate_plugin, do_action('activate_xenial/xenial-WP.php'), WP_Hook->do_action, WP_Hook->apply_filters, xwp_activation, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[31-May-2025 21:31:28 UTC] PHP Warning:  Undefined property: stdClass::$Types in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 130
[31-May-2025 21:31:28 UTC] PHP Warning:  Undefined property: stdClass::$Types in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 137
[31-May-2025 21:31:38 UTC] PHP Warning:  Undefined array key "DOCUMENT_ROOT" in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:38 UTC] PHP Warning:  include_once(/wp-config.php): Failed to open stream: No such file or directory in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:38 UTC] PHP Warning:  include_once(): Failed opening '/wp-config.php' for inclusion (include_path='.:') in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:38 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('1', NULL, NULL, NULL, NULL, NULL, NULL, NULL)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES (:param0 , NULL, NULL, NULL, NULL, NULL, NULL, NULL) | parameters: 1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 table wp_schemaLocalBusiness has no column named image.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-content/plugins/xenial/xenial-WP.php(104): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): xwp_activation(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_xenial...', false)
#7 /internal/eval.php(20): activate_plugin('xenial/xenial-W...')
#8 {main}</pre>
 for query INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('1', NULL, NULL, NULL, NULL, NULL, NULL, NULL) made by activate_plugin, do_action('activate_xenial/xenial-WP.php'), WP_Hook->do_action, WP_Hook->apply_filters, xwp_activation, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[31-May-2025 21:31:38 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('2', NULL, NULL, NULL, NULL, NULL, NULL, NULL)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES (:param0 , NULL, NULL, NULL, NULL, NULL, NULL, NULL) | parameters: 2</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 table wp_schemaLocalBusiness has no column named image.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-content/plugins/xenial/xenial-WP.php(106): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): xwp_activation(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_xenial...', false)
#7 /internal/eval.php(20): activate_plugin('xenial/xenial-W...')
#8 {main}</pre>
 for query INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('2', NULL, NULL, NULL, NULL, NULL, NULL, NULL) made by activate_plugin, do_action('activate_xenial/xenial-WP.php'), WP_Hook->do_action, WP_Hook->apply_filters, xwp_activation, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[31-May-2025 21:31:38 UTC] PHP Warning:  Undefined property: stdClass::$Types in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 130
[31-May-2025 21:31:38 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_schemaLocalBusiness` MODIFY COLUMN `pages` VARCHAR(255);</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/xenial/xenial-WP.php(131): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): xwp_Update_Database(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_xenial...', false)
#6 /internal/eval.php(20): activate_plugin('xenial/xenial-W...')
#7 {main}</pre>
 for query ALTER TABLE `wp_schemaLocalBusiness` MODIFY COLUMN `pages` VARCHAR(255); made by activate_plugin, do_action('activate_xenial/xenial-WP.php'), WP_Hook->do_action, WP_Hook->apply_filters, xwp_Update_Database, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[31-May-2025 21:31:38 UTC] PHP Warning:  Undefined property: stdClass::$Types in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 137
[31-May-2025 21:31:47 UTC] PHP Warning:  Undefined array key "DOCUMENT_ROOT" in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:47 UTC] PHP Warning:  include_once(/wp-config.php): Failed to open stream: No such file or directory in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:47 UTC] PHP Warning:  include_once(): Failed opening '/wp-config.php' for inclusion (include_path='.:') in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 73
[31-May-2025 21:31:47 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('1', NULL, NULL, NULL, NULL, NULL, NULL, NULL)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES (:param0 , NULL, NULL, NULL, NULL, NULL, NULL, NULL) | parameters: 1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 table wp_schemaLocalBusiness has no column named image.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-content/plugins/xenial/xenial-WP.php(104): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): xwp_activation(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_xenial...', false)
#7 /internal/eval.php(20): activate_plugin('xenial/xenial-W...')
#8 {main}</pre>
 for query INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('1', NULL, NULL, NULL, NULL, NULL, NULL, NULL) made by activate_plugin, do_action('activate_xenial/xenial-WP.php'), WP_Hook->do_action, WP_Hook->apply_filters, xwp_activation, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[31-May-2025 21:31:47 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('2', NULL, NULL, NULL, NULL, NULL, NULL, NULL)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES (:param0 , NULL, NULL, NULL, NULL, NULL, NULL, NULL) | parameters: 2</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 table wp_schemaLocalBusiness has no column named image.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-content/plugins/xenial/xenial-WP.php(106): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): xwp_activation(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_xenial...', false)
#7 /internal/eval.php(20): activate_plugin('xenial/xenial-W...')
#8 {main}</pre>
 for query INSERT INTO `wp_schemaLocalBusiness` (`id`, `addressLocality`, `addressRegion`, `postalCode`, `streetAddress`, `pages`, `paymentAccepted`, `image`) VALUES ('2', NULL, NULL, NULL, NULL, NULL, NULL, NULL) made by activate_plugin, do_action('activate_xenial/xenial-WP.php'), WP_Hook->do_action, WP_Hook->apply_filters, xwp_activation, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[31-May-2025 21:31:47 UTC] PHP Warning:  Undefined property: stdClass::$Types in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 130
[31-May-2025 21:31:47 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_schemaLocalBusiness` MODIFY COLUMN `pages` VARCHAR(255);</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/xenial/xenial-WP.php(131): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-includes/class-wp-hook.php(324): xwp_Update_Database(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_xenial...', false)
#6 /internal/eval.php(20): activate_plugin('xenial/xenial-W...')
#7 {main}</pre>
 for query ALTER TABLE `wp_schemaLocalBusiness` MODIFY COLUMN `pages` VARCHAR(255); made by activate_plugin, do_action('activate_xenial/xenial-WP.php'), WP_Hook->do_action, WP_Hook->apply_filters, xwp_Update_Database, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[31-May-2025 21:31:47 UTC] PHP Warning:  Undefined property: stdClass::$Types in /wordpress/wp-content/plugins/xenial/xenial-WP.php on line 137

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"xerxes-weather"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/xerxes-weather.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/xerxes-weather.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/xerxes-weather.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/xerxes-weather.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
